<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="USHYDIVENUGOP06" failures="2" tests="2" name="AddCustomerTest" time="6.334" errors="0" timestamp="19 Aug 2021 17:12:09 GMT">
  <testcase classname="com.intella.testCases.AddCustomerTest" name="AddCustomer" time="3.063">
    <failure type="java.lang.AssertionError" message="Login Test Failed">
      <![CDATA[java.lang.AssertionError: Login Test Failed
at com.intella.testCases.AddCustomerTest.AddCustomer(AddCustomerTest.java:33)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- AddCustomer -->
  <testcase classname="com.intella.testCases.AddCustomerTest" name="AddCustomer" time="3.148">
    <failure type="java.lang.AssertionError" message="Login Test Failed">
      <![CDATA[java.lang.AssertionError: Login Test Failed
at com.intella.testCases.AddCustomerTest.AddCustomer(AddCustomerTest.java:33)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- AddCustomer -->
</testsuite> <!-- AddCustomerTest -->
